orders/views.py

```diff
@@ -67,18 +68,44 @@ def get(self, request):
         except OrderItem.DoesNotExist:
             return JsonResponse({'message': 'NOTHING_IN_CART'}, status=400)
 
+class PaymentView(View):
```

Comment:
```suggestion
class OrderView(View):
```
url의 코멘트와 이어서 뷰도 payment보다는 Order가 조금 더 의미가 더 맞지 않을까 싶어요

Reply:
수정 완료했습니다!